updater

```diff
@@ -430,50 +434,84 @@ elif [ "$runmode" == cli ];then #return list of updatable apps, and ask the user
   
   updatable_apps="$(get_updatable_apps)"
   updatable_files="$(get_updatable_files)"
+  echo 
   
   if [ -z "$updatable_files" ] && [ -z "$updatable_apps" ];then
-    echo "Nothing is updatable."
+    status "Nothing is updatable."
     exit 0
   fi
   
-  echo "These apps can be updated:"
-  echo -n "$updatable_apps" | sed 's/^/  - /g'
-  echo
+  if [ -z "$updatable_apps" ];then
+    status "No apps is updatable."
+    echo
+  else
+    status "These apps can be updated:"
+    echo
+    echo "$updatable_apps" | sed 's/^/  - /g'
+    echo
+  fi
   
-  echo "These files can be updated:"
-  echo -n "$updatable_files" | sed 's/^/  - /g'
-  echo
+  if [ -z "$updatable_files" ];then
+    status "No files is updatable"
+    echo
+  else
+    status "These files can be updated:"
+    echo
+    echo "$updatable_files" | sed 's/^/  - /g'
+    echo
+  fi
   
-  read -p "Update now? [Y/n] " answer
+  question=$(status 'Update now? [Y/n] ')
```

Comment:
Not sure why you're placing the question into an array.

Reply:
> Not sure why you're placing the question into an array.

Just want to have `status` colors to the question.

Reply:
> > Not sure why you're placing the question into an array.
> 
> Just want to have `status` colors to the question.

Oh I see. This may be fine for now, but down the road I may tweak the `status` function to be more than blue text. It may prefix everything with "`Status: `", or underline the text, or something else undesirable for asking a question. Son in this case, I recommend you replace it with `echo -en` (and then the codes for the blue color) Color codes can be found [here](https://misc.flogisoft.com/bash/tip_colors_and_formatting).

Reply:
Just found that can be done like this: `read -p "$(status 'Update now? [Y/n] ')" answer`.